updating language about joss and members (#24)
pyOpenSci/pyopensci.github.io: _pages/home.md

6. **Spread the word**! pyOpenSci is a community-driven project. Tell your friends,
   recommend that a colleague submit a package, mention us at conferences, spread the word!

### Community meetings

Currently, the pyOpenSci team holds community meetings roughly every 2 weeks. These are
held remotely, and open to anyone who would like to attend. We discuss ideas for improving
the process / technology / community management / etc of the pyOpenSci community, and
would value your participation!

The best place to find information about the next community meeting is
[in the community forum](https://pyopensci.discourse.group). We post dates / times
for new meetings there.

### Who's Involved

See our [contributors page](contributors.html) for a list of the contributors
in the pyOpenSci community.

## FAQ

### What's the difference between pyOpenSci and JOSS?

The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
to improving the visibility and quality of scientific software. They do
so by providing a lightweight review and publishing process so that authors
of packages can publish their packages with a DOI and citable artifact.

> They intentionally try to avoid lengthy review and feedback cycles

Reviewer: I'm not sure this is quite right. I'd rather see this phrased something like:

"JOSS reviews are limited in scope and the [submission criteria](https://joss.readthedocs.io/en/latest/review_criteria.html) less stringent than those of pyOpenSci."

with the goal of streamlining the publishing process.

pyOpenSci encourages a more hands-on approach in the review process, and
focuses more around software best-practices than publishing.
We encourage our reviewers to provide more hands-on assistance and
give more in-depth feedback than is typically required in JOSS. The goal
of pyOpenSci is to ensure that all of its packages meet a minimum level

> of best practices and standards, and our review processes often take longer

Reviewer 2: @choldgraf the one big difference is pyopensci accepts API wrappers. JOSS does not. so pyopensci accepts MORE packages than JOSS does .

Author: cool - feel free to either make edits directly, or I can make edits when I'm back at home (out of the office hanging out w/ family today)

Reviewer 2: wonderful. ok i'll make a few edits probably after the meeting. if leo or luiz or lindsay are there, they can help review the language!!! if not i can bug @arfon who might also be willing to look at this PR and ensure we are describing JOSS in the way he'd like us to!!

in order to help authors reach this bar.

pyOpenSci has a close relationship with JOSS, and we see these two communities

> as complementary of one another. **Any package that passes the pyOpenSci review

Reviewer: It might be good to try and allude to the fact that some pyOpenSci submissions may be out of scope for JOSS:

```suggestion
as complementary of one another. **Any package that passes the pyOpenSci review and is [within scope](https://joss.readthedocs.io/en/latest/submitting.html#submission-requirements) for JOSS
```

> process is green-lit in the JOSS process** (with the exception of a few

Reviewer 3: Is it true that any package that passes pyOpenSci review is green-lit in JOSS, even if it is an API wrapper? If not, we might do something like 
```suggestion
process and is within the scope of JOSS submissions is green-lit in the JOSS process** (with the exception of a few
```

> process is green-lit in the JOSS process** (with the exception of a few

Reviewer: ```suggestion
process can be fast-tracked through the JOSS review process** (with the exception of a few
```

5. **Help with technical infrastructure**. pyOpenSci has a few pieces of technical
   infrastructure (like this site) which need maintenance and development.
6. **Spread the word**! pyOpenSci is a community-driven project. Tell your friends,
   recommend that a colleague submit a package, mention us at conferences, spread the word!

### Community meetings

Currently, the pyOpenSci team holds community meetings roughly every 2 weeks. These are
held remotely, and open to anyone who would like to attend. We discuss ideas for improving
the process / technology / community management / etc of the pyOpenSci community, and
would value your participation!

The best place to find information about the next community meeting is
[in the community forum](https://pyopensci.discourse.group). We post dates / times
for new meetings there.

### Who's Involved

See our [contributors page](contributors.html) for a list of the contributors
in the pyOpenSci community.

## FAQ

### What's the difference between pyOpenSci and JOSS?

The [Journal of Open Source Software](https://joss.theoj.org/) is a community dedicated
to improving the visibility and quality of scientific software. They do

> so by providing a lightweight review and publishing process so that authors

Reviewer 4: "lightweight" really minimizes the JOSS review process. I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation. It would be more fair for pyopensci to simply state the differences in the review process and let the reader decide whether one happens to be lightweight or not.

Reviewer: > "lightweight" really minimizes the JOSS review process. I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation. It would be more fair for pyopensci to simply state the differences in the review process and let the reader decide whether one happens to be lightweight or not.

Yeah, that's a fair point. I guess I was trying to say _lighter_ weight than pyOpenSci but I see what you mean @moorepants about potentially framing the JOSS review process in a negative light using language like this.

> I'd hope that JOSS wants reviews to be rigorous so that the journal has a decent reputation.

💯absolutely.

Author: Thanks for noticing this - I agree we definitely don't want to convey the idea that JOSS is a less-than-rigorous review process. Could somebody suggest some better wording that conveys the ideas we want?

Reviewer 4: I would remove "lightweight" as a start.

Author: https://github.com/pyOpenSci/pyopensci.github.io/pull/27 there's a start - feel free to suggest edits and I'm happy to iterate there

Reviewer 2: thanks guys!! we want everyone to be happy with how we represent our partnership on the website. all feedback is very much appreciated!!!
